### DISCUSSION TITLE
select mods button weird colours on click

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#21324] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/78441708/202888594-9a9cfc4a-3a75-461c-bbf4-b717ab617804.png)
when you click the "select mods" button it looks weird

how it normally looks:
![image](https://user-images.githubusercontent.com/78441708/202888628-80411fd7-d1d6-4da6-b580-980a5f12ce86.png)

2022.1117.0

**Comment by frenzibyte** [comment, marked as answer]
Has been fixed in https://github.com/ppy/osu/pull/21289
